### ISSUE TITLE
[Image]: QA: No label for the modal dialog

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9707]
### Description

Page Element:
lightbox that is selected from the Knobs

Issue Description:
No label for the modal dialog

Recommend Fix:
Provide appropriate label for the dialog

User Impact:
When the screen reader is unable to identify a label, it’s difficult for users to understand the purpose of the control.

Tool:
JAWS_Desktop
Voiceover_iPhone

### Component(s) impacted

Image

### Browser

Safari, Firefox

### Carbon for IBM.com version

v1.40.0-rc.0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components-react

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
